### PR TITLE
Add sample code for Azure Content Safety Video Moderation

### PR DESCRIPTION
**Pull request by g1910**
# Description

This PR adds a sample code file to run video moderation via Azure AI Content Safety. 

# All SDK Contribution checklist:
- [x] **The pull request does not introduce [breaking changes]**
- [x] **CHANGELOG is updated for new features, bug fixes or other significant changes.** (N/A: Only sample code added)
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md).**

## General Guidelines and Best Practices
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md##building-and-testing)
- [x] Pull request includes test coverage for the included changes. (N/A: Only sample code added)

### CODE REVIEW

**Comment by g1910** [comment]
@JieZhou000 could you please help in reviewing this PR? This PR adds sample code on how to use Azure AI Content Safety Service for content moderation of videos. Thanks!

**Comment by JieZhou000** [comment]
> @JieZhou000 could you please help in reviewing this PR? This PR adds sample code on how to use Azure AI Content Safety Service for content moderation of videos. Thanks!

**Comment by JieZhou000** [comment]
Please resolve the ci issue first, thanks!

File: sdk/contentsafety/azure-ai-contentsafety/samples/sample_data/video.mp4
Comment (JieZhou000) [inline sdk/contentsafety/azure-ai-contentsafety/samples/sample_data/video.mp4:1]: please double check the file you uploaded, any videos or images we upload must be copyrighted.

**Comment by JieZhou000** [comment]
Offline has communicated with our PM. The video samples should be placed in the Azure sample repository. The SDK has already been released, and updating the documentation here will no longer be updated. The link to the sample repository is: https://github.com/Azure-Samples/AzureAIContentSafety/tree/main/python/1.0.0

File: sdk/contentsafety/azure-ai-contentsafety/samples/sample_data/video.mp4
Comment (g1910) [reply sdk/contentsafety/azure-ai-contentsafety/samples/sample_data/video.mp4:1]: Thanks @JieZhou000. Can you guide me how to do this - "any videos or images we upload must be copyrighted"?

**Comment by g1910** [comment]
@JieZhou000 Thanks for the guidance. I have created a PR in the Azure Samples repo for this. https://github.com/Azure-Samples/AzureAIContentSafety/pull/15 

Could you please review it whenever possible? Thanks!

**Comment by annatisch** [comment]
> @JieZhou000 Thanks for the guidance. I have created a PR in the Azure Samples repo for this. [Azure-Samples/AzureAIContentSafety#15](https://github.com/Azure-Samples/AzureAIContentSafety/pull/15)
> 
> Could you please review it whenever possible? Thanks!

Thanks @g1910 - does that mean this PR should be closed?

**Comment by g1910** [comment]
@annatisch Yes this PR can be closed. Let me do that.

**Comment by annatisch** [comment]
Thanks @g1910!

**Pull request closed without merging**